src/api/modules/utils.ts

```diff
@@ -129,81 +128,99 @@ export const appIdFromRegistry = (app: string, majorLocator: string) => {
     .catch(handleError(app))
 }
 
-export function optionsFormatter(billingOptions: BillingOptions) {
+const FREE_BILLING_OPTIONS_TYPE = 'free'
+export const isFreeApp = (type?: string, free?: boolean) => type === FREE_BILLING_OPTIONS_TYPE || free
+
+type BillingInfo = {
+  subscription?: number
+  currency?: string
+  termsURL?: string
+  metrics?: Array<{
+    name: string
+    ranges: Range[]
+  }>
+}
+
+const chalkBillingTable = (table: any, { subscription, metrics, currency }: BillingInfo) => {
+  table.push([{ content: chalk.bold('Billable item') }, { content: chalk.bold('Pricing') }])
+
+  if (subscription) {
+    table.push([
+      {
+        content: `Subscription (monthly)`,
+      },
+      {
+        content: `${subscription} (${currency})`,
+      },
+    ])
+  }
+  metrics?.forEach(({ name, ranges }) => {
+    table.push([
+      {
+        content: name,
+      },
+      {
+        content: ranges.reduce<string>((text, { exclusiveFrom, multiplier }) => {
+          text += `${text.length > 0 ? `\n` : ''}${multiplier} (${currency}) per unit`
+          if (exclusiveFrom > 0) {
+            text += ` - for ${exclusiveFrom + 1} or more units`
+          }
+          return text
+        }, ''),
+        hAlign: 'left',
+      },
+    ])
+  })
+}
+
+const billingInfoFromPolicy = ({ currency, billing: { items } }: Policy): BillingInfo => {
+  const subscription = items.reduce<number>((sum, { fixed }) => {
+    if (fixed) {
+      sum += fixed
+    }
+    return sum
+  }, 0)
```

Comment:
```suggestion
  const subscription = items.reduce<number>(
  (sum, { fixed }) => (fixed ? sum + fixed : sum),
  0
)
```

Reply:
Nice, done in 6dcee75b3c884f4f750ee7ca20680fff37b73dff, thanks!